.github/workflows/changelog.yml

```diff
@@ -0,0 +1,42 @@
+name: Generate changelog
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        description: 'Target release version (git tag)'
+      prev_version:
+        required: true
+        description: 'Previous release version (git tag)'
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: "Generate token"
+        id: generate_token
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92
+        with:
+          app_id: ${{ secrets.GRAFANA_DELIVERY_BOT_APP_ID }}
+          private_key: ${{ secrets.GRAFANA_DELIVERY_BOT_APP_PEM }}
```

Comment:
you can do this now or later, but I think we can just use `secrets.GITHUB_TOKEN` and [set the permissions in the workflow](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs). for github releases it needs `contents: write`